Library/Formula/binwalk.rb

```diff
@@ -68,50 +53,14 @@ def install
       end
     end
 
-    if build.head?
-      ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
-      system "python", *Language::Python.setup_install_args(libexec)
-      bin.install Dir["#{libexec}/bin/*"]
-      bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
-    else
-      system "./configure", "--prefix=#{prefix}", "--disable-bundles"
-      system "make", "install"
-    end
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+    bin.install Dir["#{libexec}/bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
   end
 
   test do
     touch "binwalk.test"
-    system "#{bin}/binwalk", "binwalk.test"
+    system "#{libexec}/bin/binwalk", "binwalk.test"
```

Comment:
For some reason, this fails otherwise with
```
$ brew test binwalk
Testing binwalk
==> Using the sandbox
==> /usr/local/Cellar/binwalk/2.1.1/bin/binwalk binwalk.test
Last 15 lines from /Users/dan/Library/Logs/Homebrew/binwalk/01.binwalk:
2016-01-05 12:41:00 +0100

/usr/local/Cellar/binwalk/2.1.1/bin/binwalk
binwalk.test

Traceback (most recent call last):
  File "/usr/local/Cellar/binwalk/2.1.1/libexec/bin/binwalk", line 57, in <module>
    main()
  File "/usr/local/Cellar/binwalk/2.1.1/libexec/bin/binwalk", line 29, in main
    with binwalk.Modules() as modules:
AttributeError: __exit__
Error: binwalk: failed
Failed executing: /usr/local/Cellar/binwalk/2.1.1/bin/binwalk binwalk.test
/usr/local/Library/Homebrew/formula.rb:1473:in `block in system'
/usr/local/Library/Homebrew/formula.rb:1410:in `open'
/usr/local/Library/Homebrew/formula.rb:1410:in `system'
/usr/local/Library/Formula/binwalk.rb:64:in `block in <class:Binwalk>'
/usr/local/Library/Homebrew/formula.rb:1319:in `block in run_test'
/usr/local/Library/Homebrew/extend/fileutils.rb:17:in `mktemp'
/usr/local/Library/Homebrew/formula.rb:1315:in `run_test'
/usr/local/Library/Homebrew/test.rb:28:in `block in <main>'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/timeout.rb:66:in `timeout'
/usr/local/Library/Homebrew/test.rb:27:in `<main>'
```